### PR TITLE
Convert Header into Style Component

### PR DESCRIPTION
**Pull request by cadezacharias-hype**
### Describe your changes 
Migrates header to Styled Component
### Link the related issue
_Closes #2703_

### Instructions for Reviewer
**Functionalities or workflows that should specifically be tested.**

1. yarn start

2. Compare header to staging env

Styled Components:
image

Develop
image

Styled on Mobile:
image

Develop on Mobile:
image

### CODE REVIEW

File: src/App/components/PageHeader/Account/WalletDropdown/WalletDropdown.module.css [hunk 1]
```diff
@@ -1,78 +1,3 @@
...
-
-.actions_container {
-    width: 100%;
-    display: grid;
-    grid-template-columns: 1fr 1fr;
-    gap: 1rem;
-    bottom: 0;
-    z-index: 9;
-}
 .account_button {
```
Comment (enwong10) [inline src/App/components/PageHeader/Account/WalletDropdown/WalletDropdown.module.css:1]: why wasn't this removed?

File: src/App/components/PageHeader/Account/WalletDropdown/WalletDropdown.tsx [hunk 2]
```diff
@@ -150,42 +157,47 @@ export default function WalletDropdown(props: WalletDropdownPropsIF) {
...
-                        >
-                            <FiCopy />
-                        </button>
-                    </div>
-                    <div className={styles.wallet_display}>
-                        <p>{connectorName}</p>
-                        <p>{props.accountAddress}</p>
-                    </div>
+
+                <div>
```
Comment (enwong10) [inline src/App/components/PageHeader/Account/WalletDropdown/WalletDropdown.tsx:174]: lets get rid of the extra div here

File: src/components/Global/DropdownMenu2/DropdownMenu2.styles.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+import { motion } from 'framer-motion';
+
+export const Menu = styled.div`
```
Comment (enwong10) [inline src/components/Global/DropdownMenu2/DropdownMenu2.styles.ts:4]: for all of these styled components that contains `display: flex`, can we extend the FlexContainer class as discussed?

File: src/styled/Components/Header.ts [hunk 4]
```diff
@@ -0,0 +1,640 @@
+import { Link } from 'react-router-dom';
```
Comment (enwong10) [inline src/styled/Components/Header.ts:1]: same with this, ideally anything with `display: flex` should extend `FlexContainer`

File: src/App/components/PageHeader/Account/WalletDropdown/WalletDropdown.module.css (hunk 1 above)
Comment (enwong10) [reply src/App/components/PageHeader/Account/WalletDropdown/WalletDropdown.module.css:1]: I think it'll work if you do `styled(NavLink)`

File: src/App/components/PageHeader/Account/WalletDropdown/WalletDropdown.tsx (hunk 2 above)
Comment (cadezacharias-hype) [reply src/App/components/PageHeader/Account/WalletDropdown/WalletDropdown.tsx:174]: done

File: src/components/Global/DropdownMenu2/DropdownMenu2.styles.ts (hunk 3 above)
Comment (cadezacharias-hype) [reply src/components/Global/DropdownMenu2/DropdownMenu2.styles.ts]: done

File: src/styled/Components/Header.ts (hunk 4 above)
Comment (cadezacharias-hype) [reply src/styled/Components/Header.ts:1]: done

**Pull request merged**